kotlin/patterns-use-cases/src/main/kotlin/my/example/schedulingtasks/PaymentTracker.kt

```diff
@@ -0,0 +1,50 @@
+package my.example.schedulingtasks
+
+import dev.restate.sdk.annotation.Handler
+import dev.restate.sdk.annotation.VirtualObject
+import dev.restate.sdk.http.vertx.RestateHttpEndpointBuilder
+import dev.restate.sdk.kotlin.KtStateKey
+import dev.restate.sdk.kotlin.ObjectContext
+import dev.restate.sdk.kotlin.runBlock
```

Comment:
Do always the star import for `dev.restate.sdk.kotlin`

```suggestion
import dev.restate.sdk.kotlin.*
```